### PR TITLE
webgpu: Update README

### PR DESCRIPTION
**Pull request by qjia7**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6353)

### CODE REVIEW

**Comment by qjia7** [comment]
@gyagp @haoyunfeix @xhcao @axinging Please take a look, thanks.

File: tfjs-backend-webgpu/README.md [hunk 1]
```diff
@@ -1,13 +1,90 @@
...
+- Face landmarks detection
+- HandPose
+- MobileNet
+- PoseDetection
+- Universal sentence encoder
+- AutoML Image classification
+- AutoML Object detection
+- Speech commands
+
+Note that WebGPU 1.0 hasn't been officially released by W3C group. Currently WebGPU in chrome is in Origin Trial. To try webgpu features, you need to launch chrome canary browser with flag --enable-unsafe-webgpu.
```
Comment (gyagp) [inline tfjs-backend-webgpu/README.md:16]: Not sure about the version schema of WebGPU, we can just remove "1.0" now.

Comment (gyagp) [reply tfjs-backend-webgpu/README.md:16]: Suggest to change to this: Note that WebGPU hasn't been officially supported by browsers. Only Google Chrome is well tested, and its Canary Channel is highly recommended for your trial. Specify option "--enable-unsafe-webgpu" before you start the Chrome.

File: tfjs-backend-webgpu/README.md [hunk 2]
```diff
@@ -1,13 +1,90 @@
...
-    $ yarn # to install dependencies
-    $ yarn test
+
+### Via a script tag
+
+```html
+<!-- Import @tensorflow/tfjs or @tensorflow/tfjs-core -->
+<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs/dist/tf.min.js"> </script>
+
+<!-- Adds the WebGPU backend to the global backend registry -->
```
Comment (gyagp) [inline tfjs-backend-webgpu/README.md:38]: Adds -> Add (WASM README has issue)

File: tfjs-backend-webgpu/README.md [hunk 3]
```diff
@@ -1,13 +1,90 @@
...
+
+### When should I use the WebGPU backend?
+WebGPU as the successor of WebGL will provide a hardware accelerated solution.
+Theoretically, webgpu is recommended to be used in any scenario used by WebGL
+and provide comparable or better performance.
+
+We are committed to supporting the WebGPU backend and will continue to improve
+performance. We plan to follow the WebGPU standard closely and benefit from
+its upcoming features such as fp16, int8.
+
```
Comment (gyagp) [inline tfjs-backend-webgpu/README.md:55]: How about this: "The mission of WebGPU backend is to achieve the best performance among all approaches. However, this target can not be met overnight, but we are committed to supporting it with rapid and continuous performance improvement. Many exciting features, like FP16, DP4A, will be brought in soon."

File: tfjs-backend-webgpu/README.md [hunk 4]
```diff
@@ -1,13 +1,90 @@
...
 ```
 
-# To run the test suite:
-The `$CHROME_BIN` environment variable must be set to the location of the Chrome Canary application. 
+## Testing
+The `$CHROME_BIN` environment variable must be set to the location of the Chrome Canary application.
 
-e.g. in `~/.bash_profile`:
+e.g.
+`export CHROME_BIN="/Applications/Google Chrome Canary.app/Contents/MacOS/Google Chrome Canary"`
```
Comment (gyagp) [inline tfjs-backend-webgpu/README.md:87]: Maybe we can at least list commands for Windows, Linux and macOS.

File: tfjs-backend-webgpu/README.md [hunk 5]
```diff
@@ -1,13 +1,90 @@
...
-    $ yarn test
+
+### Via a script tag
+
+```html
+<!-- Import @tensorflow/tfjs or @tensorflow/tfjs-core -->
+<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs/dist/tf.min.js"> </script>
+
+<!-- Adds the WebGPU backend to the global backend registry -->
+<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-webgpu/dist/tf-backend-webgpu.js"></script>
```
[the remark this answers is not in the document]
Comment (qjia7) [reply tfjs-backend-webgpu/README.md:42]: Done

File: tfjs-backend-webgpu/README.md (hunk 3 above)
Comment (qjia7) [reply tfjs-backend-webgpu/README.md:56]: Done

File: tfjs-backend-webgpu/README.md (hunk 4 above)
Comment (qjia7) [reply tfjs-backend-webgpu/README.md:92]: Done

**Review by qjia7 — COMMENTED** [review]
Please take another look, thanks.

**Pull request merged**